backend/src/controllers/troopsController.js

```diff
@@ -0,0 +1,21 @@
+import { TroopsService } from '../services';
+import { getTroopsForKingdom, insertTroopForKingdom } from '../repos/troops';
```

Comment:
@kkeszthe these should be in the `TroopsRepo` class too, and either import the whole class and make an instance or export the instance from there and import that here